packages/sdk/src/conversation/interface.ts

```diff
@@ -436,7 +436,7 @@ export interface ConversationOptions {
  *    ...
  * });
  * ```
- * For details information about how to implement a BotSsoExecutionActivityHandler, please refer {@link DefaultBotSsoExecutionActivityHandler} class source code.
+ * For details information about how to implement a BotSsoExecutionActivityHandler, please refer DefaultBotSsoExecutionActivityHandler class source code.
```

Comment:
Maybe a link here will be more useful to discover the code.

Reply:
Ah, thanks for your suggestion

Reply:
Updated